packages/api/src/v3/resourceStates.ts

```diff
@@ -53,6 +53,7 @@ export interface AzureSolutionConfig extends Json {
   location: string;
   provisionSucceeded: boolean;
   needCreateResourceGroup: boolean;
+  teamsAppTenantId: string;
```

Comment:
From my understanding, the tenant id of Teams app should belong to M365 instead of Azure resource?

Reply:
The naming of AzureSolutionConfig is somehow not accurate, you can ignore it for now.

Reply:
Would it be better to add a comment here to avoid confusion from others while they read this part of code?

Reply:
This key is redundant and just to keep compatibility to current state file.